_config.yml

```diff
@@ -51,12 +58,10 @@ author:
 menu:
   - title:             Site Map
     url:               /compass/
-  - title:             ACORD
-    url:               /ACORD/
   - title:             Documentation
     url:               /docs/
-  - title:             ACORD Portal
-    url:               http://acord.tech/portal
+  - title:             Star Sailors
```

Comment:
I'm not sure about this completely. However, I'm thinking of having a Jekyll/bootstrap site that will be based off of [`Acord-Robotics/portal`](http://github.com/acord-robotics/portal) and will basically link to all our gh-pages/netlify sites, as well as the portal and the reddit. Or we could just link it to the reddit. IDK yet. As long as everything is integrated (integromat) everything is fine